### ISSUE TITLE
CRUD?

### ISSUE THREAD

**Issue by DaleEMoore** [fyne-io/7guis#2]
I'm DELIGHTED to find fyne is working the 7 GUIs [tasks](https://eugenkiss.github.io/7guis/tasks)! I'm especially excited anticipating the answers to the [CRUD task](https://eugenkiss.github.io/7guis/tasks#crud).

You go folks!!

**Comment by andydotxyz** [comment]
Thanks @DaleEMoore I think that the CRUD will need to wait until we have the data API available which will be in the next big release.
Watch this space :)

**Comment by DaleEMoore** [comment]
Is there someplace I could watch smaller milestones (inchstones?) as development on the next big release progresses? Or should I follow https://github.com/fyne-io/7guis/projects as tasks workflow?

**Comment by andydotxyz** [comment]
You can see the work towards next release at https://github.com/fyne-io/fyne/milestone/7.

This repository is being developed as and when we can so doesn't really have a plan of it's own.

**Comment by andydotxyz** [comment]
We moved out the data binding work to the 2.0 milestone, but this is now under way https://github.com/fyne-io/fyne/milestone/6.

If you're interested in progress you can use the Fyne `develop` branch which now has the starts of data binding and animation :)

**Comment by Jacalz** [comment]
Just linking to https://github.com/fyne-io/7guis/pull/12 as it seems to solve this.

**Comment by DaleEMoore** [comment]
This is lovely, thanks!!

All I had to do was:

   1. Look around https://github.com/fyne-io/7guis/pull/12 and find
   references to https://github.com/andydotxyz/7guis/tree/feature/crud.
   2. git clone https://github.com/andydotxyz/7guis/tree/feature/crud
   3. cd crud/crud
   4. fyne-cross android -app-id crud.mooreworks.net . >t1 2>&1

And CRUD is running on my andi, YEA!

I'll bet crud will look good at https://github.com/fyne-io/7guis sometime.

**Comment by andydotxyz** [comment]
Yup, the PR will be merged soon I hope, once I get a chance to respond to the comments.

**Comment by DaleEMoore** [comment]
Only 5 years later and I find this lovely AI write up:

https://grok.com/chat/9c04fe5e-d30b-474f-847c-ee1d7b80c444

**Comment by andydotxyz** [comment]
Oh wow, this should have been closed years ago - when that crud example was merged into the repo
